### PR TITLE
Clean up, readme update

### PR DESCRIPTION
**Pull request by maciejburda**
Minor UI tweaks and readme update.

### CODE REVIEW

File: alps-ios-TicketingApp/AppDelegate.swift [hunk 1]
```diff
@@ -21,8 +21,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         setupAppearance()
         
         // Basic setup
-        MatchMore.apiKey = "eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9.eyJpc3MiOiJhbHBzIiwic3ViIjoiYjk3MDFjNzQtMzkyMi00MjY1LTg2Y2ItNzQ0MTE4YWI4NWU0IiwiYXVkIjpbIlB1YmxpYyJdLCJuYmYiOjE1MTA4NDY5NTgsImlhdCI6MTUxMDg0Njk1OCwianRpIjoiMSJ9.IDnqpc3Oco3QthoCwf-sYx_3zXFaJ_h9nFzNi5hAHzUSkVlCOHavxdBzQYeBnak744O4nr7dZHr1hb_P3YzoPQ"
-        MatchMore.worldId = "b9701c74-3922-4265-86cb-744118ab85e4"
+        MatchMore.apiKey = "eyJ0eXAiOiJKV1QiLCJhbGciOiJFUzI1NiJ9.eyJpc3MiOiJhbHBzIiwic3ViIjoiZDY1YjIwYTAtMjAwZS00MzE4LWEyZjAtNTdkZGU1ZDE0YTJiIiwiYXVkIjpbIlB1YmxpYyJdLCJuYmYiOjE1MTExODI3NzMsImlhdCI6MTUxMTE4Mjc3MywianRpIjoiMSJ9.J7PpSnL80VG5G1QmJlzEpTLBgr0cKu0EwZaQnha07YZU135NlEI6yldUSR95md4o8liqeHyQXUqzgjWFgt-VQg"
+        MatchMore.worldId = "d65b20a0-200e-4318-a2f0-57dde5d14a2b"
```
Comment (wenjdu) [inline alps-ios-TicketingApp/AppDelegate.swift]: We should maybe commit for the Master without apikey and worldId, what do you think ? 

If we leave it, I also think it is a good "example" so they can recognize which is apikey and which is worldid

**Review by wenjdu — APPROVED** [review]

File: alps-ios-TicketingApp/AppDelegate.swift (hunk 1 above)
Comment (maciejburda) [reply alps-ios-TicketingApp/AppDelegate.swift]: I was wondering about that.

For sure they can pull them from history anyway.

I think we should keep it for ease of use sake and limit number of devices on them.

@tharpa what do you think?

Comment (tharpa) [reply alps-ios-TicketingApp/AppDelegate.swift]: That's a very good point.  We could remove them from the master and delete those worlds from production DB, so even if they get it from history they won't be valid ids and tokens.  We want everybody using us in this phase to go to the dev portal first.

So, in conclusion, I would ask you to remove them from the master.

Comment (maciejburda) [reply alps-ios-TicketingApp/AppDelegate.swift]: 👍

**Pull request merged**